### DISCUSSION TITLE
Stop forcing keypresses into the "Search or Jump to..." input controls.

### DISCUSSION THREAD

**Discussion by StingyJack** [community/community#18927] [Other Feature Feedback, Questions, & Ideas]
I didnt click in that input box, so dont send my keypresses there. This is incredibly frustrating when I have scrolled down the page and that box is no longer visible yet any stray keypress scrolls me all the way back to the top of the page. 

If I wanted to search, I would click in the input box. You are not making GH any easier to use with this, it just wastes my time making me try to find where I was a moment ago.

I dont know why you would even want to bring attention to a search that is not good or even serviceable (it cant find "DogsAndCats" if I search for "Dog" or "Cat", or "APIHelper.PerformDecryption" if I search for "APIHelper.Perf" or "decry")

**Comment by Tran-Quyen** [comment]
You may use library for your search engine

**Reply by StingyJack** [reply]
Hi, I dont know what you mean by "use library".  Can you link me to an example?

**Reply by Tran-Quyen** [reply]
> Hi, I dont know what you mean by "use library". Can you link me to an example?

May I know wrong about question?

**Reply by StingyJack** [reply]
I dont know if you are trying to tell me 
- about a code library that exists that can search git hub repos that returns more reliable results than the built in search
- that I should go to a physical library where there are books
- or something else

This feedback is about a poor UX, not the searching functionality (I've reported that elsewhere).
